src/vs/workbench/contrib/audioCues/browser/audioCueLineFeatureContribution.ts

```diff
@@ -210,7 +213,7 @@ class MarkerLineFeature implements LineFeature {
 
 class FoldedAreaLineFeature implements LineFeature {
 	public readonly audioCue = AudioCue.foldedArea;
-
+	public readonly event = AccessibleNotificationEvent.Folded;
```

Comment:
This is fishy that every LineFeature now has an `event` and `audioCue` and that the `audioCue` can be derived from the `event`.